doc/explanation/storage.md

```diff
@@ -156,6 +156,13 @@ Each storage volume uses one of the following content types:
   Custom storage volumes of content type `block` can only be attached to virtual machines.
   They should not be shared between instances, because simultaneous access can lead to data corruption.
 
+`iso`
+: The content type is used for custom ISO volumes.
+  A custom storage volume of type `iso` can only be created by importing an ISO file using `lxc import`.
+
+  Custom storage volumes of content type `iso` can only be attached to virtual machines.
+  They can be attached to multiple machines simultaneously as they are always mounted read-only.
```

Comment:
They are technically not mounted. 

So perhaps just:

> as they are always read-only.

Reply:
Good point.